concepticondata/conceptlists/Johansson-2020-344.tsv

```diff
@@ -0,0 +1,345 @@
+Johansson-2020-344-152	152	NOSE	1221	NOSE
+Johansson-2020-344-153	153	NOT	1240	NOT
+Johansson-2020-344-154	154	NOW	1376	NOW
+Johansson-2020-344-155	155	OLD (AN)	2112	OLD (AGED)
+Johansson-2020-344-156	156	OLD (INAN)	2113	OLD (USED)
+Johansson-2020-344-157	157	OLD MAN	406	OLD MAN
+Johansson-2020-344-158	158	OLD WOMAN	907	OLD WOMAN
+Johansson-2020-344-159	159	ONE	1493	ONE
+Johansson-2020-344-160	160	OTHER	197	OTHER
+Johansson-2020-344-161	161	OUT(SIDE)	762	OUTSIDE
+Johansson-2020-344-162	162	PART	779	PART
+Johansson-2020-344-163	163	PATH	2252	PATH
+Johansson-2020-344-164	164	PENIS	1222	PENIS
+Johansson-2020-344-165	165	PERSON	683	PERSON
+Johansson-2020-344-166	166	POINTY	372	POINTED
+Johansson-2020-344-167	167	QUICK	1631	FAST
+Johansson-2020-344-168	168	QUIET	1314	QUIET
+Johansson-2020-344-169	169	RAIN	658	RAIN (PRECIPITATION)
+Johansson-2020-344-170	170	RAW	1959	RAW
+Johansson-2020-344-171	171	RED	156	RED
+Johansson-2020-344-172	172	RIGHT	1019	RIGHT
+Johansson-2020-344-173	173	RIPE	178	RIPE
+Johansson-2020-344-174	174	RIVER	666	RIVER
+Johansson-2020-344-175	175	ROOT	670	ROOT
+Johansson-2020-344-176	176	ROPE	1218	ROPE
+Johansson-2020-344-177	177	ROTTEN	1728	ROTTEN
+Johansson-2020-344-178	178	ROUGH	1923	ROUGH
+Johansson-2020-344-179	179	ROUND	1395	ROUND
+Johansson-2020-344-180	180	RUN	1519	RUN
+Johansson-2020-344-181	181	SAD	699	SAD
+Johansson-2020-344-182	182	SALTY	1091	SALTY
+Johansson-2020-344-183	183	SAME	200	SAME
+Johansson-2020-344-184	184	SAND	671	SAND
+Johansson-2020-344-185	185	SAY	1458	SAY
+Johansson-2020-344-186	186	SEA	1474	SEA
+Johansson-2020-344-187	187	SEE	1409	SEE
+Johansson-2020-344-188	188	SEED	714	SEED
+Johansson-2020-344-189	189	SEMEN	2283	SEMEN
+Johansson-2020-344-190	190	SEVEN	1704	SEVEN
+Johansson-2020-344-191	191	SHADOW	2275	SHADOW
+Johansson-2020-344-192	192	SHALLOW	193	SHALLOW
+Johansson-2020-344-193	193	SHARP	1396	SHARP
+Johansson-2020-344-194	194	SHORT	1645	SHORT
+Johansson-2020-344-195	195	SIT	1416	SIT
+Johansson-2020-344-196	196	SIX	1703	SIX
+Johansson-2020-344-197	197	SKIN	763	SKIN
+Johansson-2020-344-198	198	SKY	1732	SKY
+Johansson-2020-344-199	199	SLEEP	1585	SLEEP
+Johansson-2020-344-200	200	SLOW	701	SLOW
+Johansson-2020-344-201	201	SMALL	1246	SMALL
+Johansson-2020-344-202	202	SMELL	2124	SMELL
+Johansson-2020-344-203	203	SMOKE	778	SMOKE (EXHAUST)
+Johansson-2020-344-204	204	SMOOTH	1234	SMOOTH
+Johansson-2020-344-205	205	SNEEZE	1621	SNEEZE
+Johansson-2020-344-206	206	SNORE	1983	SNORE
+Johansson-2020-344-207	207	SOFT	1856	SOFT
+Johansson-2020-344-208	208	SOME	1241	SOME
+Johansson-2020-344-209	209	SOUR	1906	SOUR
+Johansson-2020-344-210	210	SPIT	1440	SPIT
+Johansson-2020-344-211	211	STAND	1442	STAND
+Johansson-2020-344-212	212	STAR	1430	STAR
+Johansson-2020-344-213	213	STONE	857	STONE
+Johansson-2020-344-214	214	STRAIGHT	1404	STRAIGHT
+Johansson-2020-344-215	215	STRONG	785	STRONG
+Johansson-2020-344-216	216	SUCK	1421	SUCK
+Johansson-2020-344-217	217	SUN	1343	SUN
+Johansson-2020-344-218	218	SWEAT	125	SWEAT (SUBSTANCE)
```

Comment:
If the original list does not make it clearer, I would go for the more general 2458 PERSPIRE OR SWEAT

Reply:
Yes the author lists a number of body-related verbs that were not included in the list (and 'to sweat' is among them). Another reason to map the gloss to SWEAT (SUBSTANCE) is the extensive use of Swadesh's concepts.